$NetBSD: patch-configure,v 1.1 2018/10/02 12:05:35 adam Exp $

Portablity fixes.
Disable ucontext.h detection, which leads to problems on some platforms.
Disable univeral binary build on Mac OS X.

--- configure.orig	2018-08-10 01:35:09.000000000 +0000
+++ configure
@@ -15253,7 +15253,7 @@ else
 fi
 
 
-if test "${cxxstd}" == 17 ; then
+if test "${cxxstd}" = 17 ; then
       ax_cxx_compile_alternatives="17 1z"    ax_cxx_compile_cxx17_required=true
   ac_ext=cpp
 ac_cpp='$CXXCPP $CPPFLAGS'
@@ -16966,7 +16966,7 @@ $as_echo "$as_me: WARNING: C++17 is not 
 
 	CXXFLAGS="$CXXFLAGS -std=c++17"
 else
-  if test "${cxxstd}" == 14 ; then
+  if test "${cxxstd}" = 14 ; then
         ax_cxx_compile_alternatives="14 1y"    ax_cxx_compile_cxx14_required=true
   ac_ext=cpp
 ac_cpp='$CXXCPP $CPPFLAGS'
@@ -17874,7 +17874,7 @@ $as_echo "#define HAVE_CXX14 1" >>confde
 
   	CXXFLAGS="$CXXFLAGS -std=c++14"
   else
-    if test "${cxxstd}" == 11 ; then
+    if test "${cxxstd}" = 11 ; then
         ax_cxx_compile_alternatives="11 0x"    ax_cxx_compile_cxx11_required=true
   ac_ext=cpp
 ac_cpp='$CXXCPP $CPPFLAGS'
@@ -18542,7 +18542,7 @@ $as_echo "#define HAVE_CXX11 1" >>confde
 
       CXXFLAGS="$CXXFLAGS -std=c++11"
     else
-      if test "${cxxstd}" == 03 ; then
+      if test "${cxxstd}" = 03 ; then
         $as_echo "#define ILMBASE_FORCE_CXX03 1" >>confdefs.h
 
         CXXFLAGS="$CXXFLAGS -std=c++03"
@@ -21827,11 +21827,11 @@ $as_echo "#define HAVE_CXX17 1" >>confde
   { $as_echo "$as_me:${as_lineno-$LINENO}: WARNING: C++17 is not yet standardized, so the checks may change in incompatible ways anytime" >&5
 $as_echo "$as_me: WARNING: C++17 is not yet standardized, so the checks may change in incompatible ways anytime" >&2;}
 
-        if test "$HAVE_CXX14" == 1 ; then
+        if test "$HAVE_CXX14" = 1 ; then
   	      CXXFLAGS="$CXXFLAGS -std=c++14"
           cxxstd = 14
         else
-          if test "$HAVE_CXX11" == 1 ; then
+          if test "$HAVE_CXX11" = 1 ; then
   	        CXXFLAGS="$CXXFLAGS -std=c++11"
             cxxstd = 11
           fi
@@ -22447,7 +22447,7 @@ $as_echo "#define STDC_HEADERS 1" >>conf
 
 fi
 
-for ac_header in limits.h unistd.h ucontext.h
+for ac_header in limits.h unistd.h
 do :
   as_ac_Header=`$as_echo "ac_cv_header_$ac_header" | $as_tr_sh`
 ac_fn_c_check_header_mongrel "$LINENO" "$ac_header" "$as_ac_Header" "$ac_includes_default"
@@ -22768,7 +22768,7 @@ LIB_SUFFIX=""
 lib_suffix_valid="no"
 
 lib_namespace=""
-if test "x${library_namespace_versioning}" == xyes ; then
+if test "x${library_namespace_versioning}" = xyes ; then
     cat >>confdefs.h <<_ACEOF
 #define IMATH_INTERNAL_NAMESPACE Imath_${ILMBASE_VERSION_API}
 _ACEOF
@@ -22787,7 +22787,7 @@ _ACEOF
     lib_namespace="${ILMBASE_VERSION_API}"
     LIB_SUFFIX="${ILMBASE_VERSION_API}"
     lib_suffix_valid="yes"
-elif test "x${library_namespace_versioning}" == xno ; then
+elif test "x${library_namespace_versioning}" = xno ; then
     cat >>confdefs.h <<_ACEOF
 #define IMATH_INTERNAL_NAMESPACE Imath
 _ACEOF
@@ -22823,7 +22823,7 @@ _ACEOF
 fi
 
 
-if test "x${lib_suffix_valid}" == xyes ; then
+if test "x${lib_suffix_valid}" = xyes ; then
 LIB_SUFFIX_DASH="-${LIB_SUFFIX}"
 
  if true; then
@@ -22858,7 +22858,7 @@ else
 fi
 
 
-if test "x${custom_public_namespace}" == xyes ; then
+if test "x${custom_public_namespace}" = xyes ; then
     { $as_echo "$as_me:${as_lineno-$LINENO}: WARNING: Enabling 'custom user namespace' requires an additional argument, reverting to default" >&5
 $as_echo "$as_me: WARNING: Enabling 'custom user namespace' requires an additional argument, reverting to default" >&2;}
     $as_echo "#define IMATH_NAMESPACE Imath" >>confdefs.h
@@ -22870,7 +22870,7 @@ $as_echo "$as_me: WARNING: Enabling 'cus
 
     public_namespace=""
     custom_public_namespace=no
-elif test "x${custom_public_namespace}" == xno ; then
+elif test "x${custom_public_namespace}" = xno ; then
     $as_echo "#define IMATH_NAMESPACE Imath" >>confdefs.h
 
     $as_echo "#define IEX_NAMESPACE Iex" >>confdefs.h
@@ -22911,7 +22911,7 @@ else
 fi
 
 
-  if test "${osx_arch}" != default ; then
+  if false ; then
           osx_arch_flags="-arch ${osx_arch// / -arch }"
      CXXFLAGS="$CXXFLAGS $osx_arch_flags"
 
